docs/developer-guide.md

```
@@ -54,17 +54,19 @@ spec:
         name: image-repository/tag
       name: tag-name
 ```
-### **Available annotations**  
+### **Available annotations**
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)